ntbs-integration-tests/NotificationPages/ClinicalDetailsPageTests.cs

```diff
@@ -294,7 +318,6 @@ public async Task ValidateClinicalDetailsProperties_ReturnsErrorIfBothTreatments
 
             // Assert check just response.Content
             var result = (await response.Content.ReadAsStringAsync());

```

Comment:
Redundant parentheses here (and a couple other places using the same pattern) can we please strip them out?